### ISSUE TITLE
The application does not go into the main window at first login on ubuntu 16.04.

### ISSUE THREAD

**Issue by frontendwizard** [arkenthera/Chiika#24]
After firing the application through the `/usr/share/chiika/Chiika` script (#23) and going into the login page, when I click on login the button goes gray and nothing happens.

**Comment by 0x61726b** [comment]
Can you find the `chiika.log` file and post here? Should be in `/home/user/.config/chiika`.

**Comment by frontendwizard** [comment]
Here it goes.

[chiika.txt](https://github.com/arkenthera/Chiika/files/482572/chiika.txt)

**Comment by 0x61726b** [comment]
From the looks of it, login is succesful but it just didnt create the main window after the login returned. Did you try launching Chiika again?

**Comment by frontendwizard** [comment]
You are right, I've opened the application again and this time went directly into the Homepage.

**Comment by 0x61726b** [comment]
I'll look into why it didnt go to home page after the first login, so keeping the issue open for now.
